src/Sarif.WorkItemFiling/Extensions.cs

```diff
@@ -50,7 +50,7 @@ public static WorkItemFilingMetadata CreateWorkItemFilingMetadata(this SarifLog
 
                 foreach (string fieldKvp in fieldKvps)
                 {
-                    string[] kv = fieldKvp.Split(':');
+                    string[] kv = fieldKvp.Split(new char[] { ':' }, 2);
 
                     if (kv.Length == 2)
```

Comment:
do we need this check anymore?

Reply:
Yes because it will split at most 2 but could be less.